scripts/eslint/rules/imports.js

```
@@ -53,6 +53,28 @@ function makeMuiTableRules() {
+    name: 'apollo-upload-client/createUploadLink.mjs',
+    message:
+      'apollo-upload-client has been removed. Use HttpLink from @apollo/client instead. All file uploads must use the useMinioUpload hook with presigned URLs.',
+  },
+  {
+    id: 'apollo-upload-client-link-js',
+    name: 'apollo-upload-client/createUploadLink.js',
+    message:
+      'apollo-upload-client has been removed. Use HttpLink from @apollo/client instead. All file uploads must use the useMinioUpload hook with presigned URLs.',
+  },
+  {
+    id: 'apollo-upload-client-link-bare',
```

Biome (2.4.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)